src/pages/scheduler/components/ShareTimetableButton.tsx

```diff
@@ -65,21 +65,27 @@ const ShareCourseCard = (props: { course: SavedCourse }) => {
 const SelectedCoursesTable = (props: { term: string }) => {
   const { courses } = useSavedCourses();
 
+  const inSession_savedCourses = courses
+    .filter((course) => course.term === props.term)
+    .filter((course) => course.lecture != null || course.lab != null || course.tutorial != null);
```

Comment:
Right idea but you can just `or` everything instead.

Reply:
you mean by taking out `!=null`? I thought it [wouldn't work](https://stackoverflow.com/questions/20940545/how-can-i-check-whether-an-optional-parameter-was-provided) for both `null` and `undefined` values if I didn't include that?